### PR TITLE
Replace viscoplastic parameter with Enable elasticity

### PR DESCRIPTION
**Pull request by anne-glerum**
Instead of an additional parameter in the viscoplastic material model, use the parameter `Enable elasticity` in subsection Formulation to determine whether elasticity should be switched on. `Enable elasticity` needs to be true in any case to use elasticity in the viscoplastic material model. 

### For all pull requests:

* [x] I have followed the [instructions for indenting my code](../blob/main/CONTRIBUTING.md#making-aspect-better).

### For new features/models or changes of existing features:

* [ ] I have tested my new feature locally to ensure it is correct.
* [ ] I have [created a testcase](https://aspect-documentation.readthedocs.io/en/latest/user/extending/testing/writing-tests.html) for the new feature/benchmark in the [tests/](../blob/main/tests/) directory.
* [x] I have added a changelog entry in the [doc/modules/changes](../blob/main/doc/modules/changes) directory that will inform other users of my change.

### CODE REVIEW

File: source/material_model/rheology/visco_plastic.cc [hunk 1]
```diff
@@ -617,7 +613,7 @@ namespace aspect
         friction_models.initialize_simulator (this->get_simulator());
         friction_models.parse_parameters(prm);
 
-        use_elasticity = prm.get_bool ("Include viscoelasticity");
+        use_elasticity = this->get_parameters().enable_elasticity;
```
Comment (gassmoeller) [inline source/material_model/rheology/visco_plastic.cc:616]: Can you either eliminate the whole variable and replace with calls to this->get_parameters().enable_elasticity (they are cheap), or rename the variable to `enable_elasticity`?

**Review by gassmoeller — COMMENTED** [review]
Looks good, but this will require changes to parameter files and should therefore also get a changelog entry.

**Review by bangerth — APPROVED** [review]
If the old and new parameter had been in the same section, we could have declared them as aliases: https://dealii.org/developer/doxygen/deal.II/classParameterHandler.html#aecb847561585089c4c7fcf51eddece16 But this doesn't work with parameters in different sections.

**Comment by anne-glerum** [comment]
> If the old and new parameter had been in the same section, we could have declared them as aliases: https://dealii.org/developer/doxygen/deal.II/classParameterHandler.html#aecb847561585089c4c7fcf51eddece16 But this doesn't work with parameters in different sections.

Good to know!

**Pull request merged**